angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/weather.html

```diff
@@ -26,13 +37,16 @@
                         <div class="col-xs-3 day">
                           <p>Now</p>
                           <img ng-src="{{config.iconPrefix}}{{weather.currentWeather.imgName}}{{config.iconPostfix}}" title="{{weather.currentWeather.condition}}" alt="{{weather.currentWeather.condition}}">
-                          <p>{{weather.currentWeather.temperature}}&deg; {{weather.temperatureUnit}}</p>
+                          <p ng-show="!showMetric" >{{weather.currentWeather.temperature | number:0}}&deg; {{weather.temperatureUnit}}</p>
```

Comment:
The call to action might be weird. What about just displaying both at the same time?